typings/glamorous.test.tsx

```diff
@@ -0,0 +1,21 @@
+import glamorous from "../";
+
```

Comment:
Hmmm.... So is this file not run at all? Sorry, I'm just confused about what purpose this file serves